modules/mcuboot/CMakeLists.txt

```diff
@@ -489,13 +489,27 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
         )
     else()
```

Comment:
Please fix commit message
`mcuboot: cmake: Add DFU zip variant for MCUboot Direct-XIP`
This prevents confusing it with nRF53 multi-image update